### ISSUE TITLE
[ALM Accelerator - QUESTION] How to bypass approval for test environment but approval will be mandatory for prod environnment

### ISSUE THREAD

**Issue by tocri** [microsoft/coe-starter-kit#5953]
### What is your question?

Hi,

I know i can use bypass approval to skip the approval when i want to deploy a solution to TEST and PROD. But is there a way to bypass approval only for the test environment only ? 

Thanks

### What component are you experiencing the issue with?

ALM Accelerator Canvas App

### What solution version are you using?

3.23052.29

**Comment by mikefactorial** [comment]
@tocri this isn't possible today from a user interface standpoint today, but I do see how this would be a useful feature. However, the bypass approval really only sets the Auto Complete flag on the PR. So, if you have a branch policy in effect on the main branch (e.g. Require 1 Approver) the PR won't be auto completed until an approver approves the PR so you can enforce the approval using a branch policy on the main branch.

**Comment by tocri** [comment]
You're right. It's exactly what i did this morning and i'm so happy because it's working well. at the end, i will improve my skill in devops...i knew nothing last week. Used only wiki and board.

![image](https://github.com/microsoft/coe-starter-kit/assets/10110260/fface1ad-94da-4600-b256-e5f7779be7a6)

<img width="507" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/10110260/d86455da-44fd-4587-8686-446b15466ccb">

It would be great to embed this functionnality into the app. I appreciate your support. thanks a lot.

**Comment by Ayesha-320** [comment]
Hi @tocri, 
According to https://learn.microsoft.com/en-us/power-platform/alm/pipelines, Power Platform Pipelines are now the recommended approach for Application Lifecycle Management (ALM). The ALM Accelerator is no longer actively maintained or supported. Therefore, this issue is being closed accordingly.  

Best Regards, 
CoE Starter Kit Team
